config-bootstrapper/src/main/java/org/hypertrace/core/bootstrapper/ConfigBootstrapper.java

```diff
@@ -19,16 +19,16 @@
 
   public static void main(String[] args) {
     updateRuntime();
-    bootstrapper(args);
+    BootstrapArgs bootstrapArgs = BootstrapArgs.from(args);
+    bootstrapper(bootstrapArgs).execute(bootstrapArgs);
   }
 
-  private static void bootstrapper(String[] args) {
-    BootstrapArgs bootstrapArgs = BootstrapArgs.from(args);
+  public static BootstrapRunner bootstrapper(BootstrapArgs bootstrapArgs) {
```

Comment:
add a one-line javadoc or comment why this is public (same rationale as this PR really)

Reply:
reason is otherwise someone can un-public it seeing visibility as a bug and cause chaos :D

Reply:
Addressed.